src/GeoJSON.jl

```diff
@@ -20,16 +20,20 @@ julia> GeoJSON.parse("{\"type\": \"Point\", \"coordinates\": [30, 10]}")
 GeoInterface.Point([30.0, 10.0])
 ```
 """
-parse(input; kwargs...) = dict2geo(JSON.parse(input; kwargs...))
+parse(input) = dict2geo(JSON3.read(input))
```

Comment:
Can we rename this as `read(input)`, deprecate `GeoJSON.parse(input)`, and point people to `GeoJSON.read(input)` instead (for consistency with `JSON3`).